docs/Tutorials/TLS.md

```diff
@@ -79,86 +47,20 @@ is also specified in the configuration file for the `nodeurl` and `clienturl` op
     tlsclienttrust = "ca"
     tlsservertrust = "ca"
 
+    clientconnectiontls = "strict"
+    clientconnectiontlsservercert = "<PATH-TO>/orion/bin/orion_cer.pem"
+    clientconnectiontlscertchain = []
+    clientconnectiontlsserverkey = "<PATH-TO>/orion/bin/orion_cer.key"
+    clientconnectiontlsservertrust = "whitelist"
+    clientconnectiontlsknownclients = "<PATH-TO>/orion/bin/orionKnownClients"
+    
     nodenetworkinterface = "0.0.0.0"
     clientnetworkinterface = "0.0.0.0"
     ```
 
-### IP address is CN 
-
-To use a public IP address as CN:
-
-#### Updating the `openssl.cfn` file
-
-1. Find the `openssl.cfn` file, and create a copy of it.
-1. In your copy of the `openssl.cfn` file, find the `[req]` section, and add:
-
-    ```
-    req_extensions = v3_req
-  
-    [ v3_req ]
-    basicConstraints = CA:FALSE
-    keyUsage = nonRepudiation, digitalSignature, keyEncipherment
-    subjectAltName = @alt_names
-
-    [alt_names]
-    DNS.1 = <DNS-PUBLIC-RECORD>
-    DNS.2 = <DNS-PRIVATE-RECORD>
-    IP.1 = <PUBLIC-IP-ADDRESS>
-    IP.2 = <PRIVATE-IP-ADDRESS>
-    ```
+## Orion-to-Orion TLS Properties 
 
-1. For each DNS you want to use as an alternate name, specify a DNS._n_ entry.
-1. For each IP address you want as an alternate IP address, specify an IP._n_ entry.
-
-#### Generating a new CSR for a node
-
-1. Run the following command. Substitute your values for all variables.
-
-    `openssl req -new -key orion_cer.key -out orion_cer.csr -config <PATH-TO>/openssl.cnf`
-
-1. Test whether the certificate was generated with the expected subject alternative names:
-
-    `openssl req -text -noout -in orion_cer.csr`
-
-    !!! Example "Example of command output"
-
-        ```
-        [...]
-        Requested Extensions:
-              X509v3 Subject Alternative Name:            
-        DNS:<DNS-PUBLIC-RECORD>,
-        DNS:<DNS-PRIVATE-RECORD>, 
-        IP Address:<PUBLIC-IP-ADDRESS>, 
-        IP Address:<PRIVATE-IP-ADDRESS>
-        [...]
-        ```
-
-#### Generating a new certificate
-
-1. Run the following command. Substitute your values for all variables.
-
-    `openssl x509 -req -in orion_cer.csr -CA orion_ca.pem -CAkey orion_ca.key -CAcreateserial -out orion_cer.pem -days 500 -sha256 -extfile <PATH-TO>/openssl.cnf -extensions v3_req`
-
-1. Test whether the generated certificate contains the subject alternative names:
-
-    `openssl x509 -in orion_cer.pem -text -noout`
-
-    !!! Example "Example of command output"
-
-        ```
-        [...]
-        X509v3 extensions:
-            X509v3 Subject Alternative Name:
-        DNS:<DNS-PUBLIC-RECORD>,
-        DNS:<DNS-PRIVATE-RECORD>, 
-        IP Address:<PUBLIC-IP-ADDRESS>, 
-        IP Address:<PRIVATE-IP-ADDRESS>
-        [...]
-        ```
-
-## TLS Properties in Orion Configuration File 
-
-TLS properties are specified in the [configuration file](../Reference/Configuration-File.md). 
+Properties to configure TLS between Orion nodes.
 
 ### tls 
```

Comment:
Current description for `off` is: Mutually authenticated TLS is not used for in- and outbound connections.

This needs updating to talk about Orion to Orion connections rather than inbound and outbound connections.

Reply:
Done.